test/e2e/petset.go

```diff
@@ -216,6 +216,10 @@ var _ = framework.KubeDescribe("PetSet [Slow] [Feature:PetSet]", func() {
 			By("Creating foo:bar in member with index 0")
 			pet.write(0, map[string]string{"foo": "bar"})
 
+			// Wait for the write to complete to all slaves with a timeout of 100s (specified in ms).
+			By("waiting for all slaves to sync")
+			pet.waitForReplication(0, ps.Spec.Replicas - 1, 100000)
```

Comment:
I see, we don't want application-specific code here? Then I guess I could make a polling mechanism that would work for all of them.